tox.ini

```diff
@@ -87,7 +88,9 @@ deps =
     # The devdeps factor is intended to be used to install the latest developer version
     # or nightly wheel of key dependencies.
     # Do not install asdf-astropy so we can test astropy.io.misc.asdf until we remove it.
-    devdeps,mpldev: git+https://github.com/matplotlib/matplotlib.git#egg=matplotlib
+    devdeps: scipy>=0.0.dev0
+    devdeps: numpy>=0.0.dev0
+    devdeps,mpldev: matplotlib>=0.0.dev0
```

Comment:
@Cadair , this picked up matplotlib 3.6.0rc2 , not a dev version. Is this supposed to happen?

Reply:
hmmm where?

Reply:
https://github.com/astropy/astropy/actions/runs/3053254195/jobs/4923644273 (the job on main after merging this)